recipes/mcmc-sampling-emcee/mcmc_sampling.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        # Update model parameters factors inplace
</pre><p>Can you add a description of the parameters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Agreed! And if it really is factor, then I would suggest to rename the <code>pars</code> to <code>factors</code> for consistency.</p><p> </p><p>Adding a docstring to this would be really helpful.</p><p>Then you could mention all the comments inside the docstring instead  </p>